hive-metastore/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -258,7 +268,7 @@ private Table newHmsTable() {
   }
 
   private void setHmsTableParameters(String newMetadataLocation, Table tbl, Map<String, String> icebergTableProps,
-      boolean hiveEngineEnabled) {
+      Set<String> obsoleteProps, boolean hiveEngineEnabled) {
```

Comment:
nit: I know it is not very consistent throughout the codebase but I think the following formatting would be more aligned with the rest of the code:

![image](https://user-images.githubusercontent.com/6235869/108401585-57cba700-71d1-11eb-9155-129ea47877d1.png)

Reply:
Makes sense, adjusted it